control/metrics.go

```diff
@@ -75,8 +66,15 @@ func errorMetricNotFound(ns string, ver ...int) error {
 	return fmt.Errorf("Metric not found: %s", ns)
 }
 
-func errorMetricContainsNotAllowedChars(ns string) error {
-	return fmt.Errorf("Metric namespace %s contains not allowed characters. Avoid using %s", ns, listNotAllowedChars())
+func errorMetricsNotFound(ns string, ver ...int) error {
```

Comment:
Do you mean  "errorMetricNotFound" and "errorMetricsNotFound" and relationship between them?

- `errorMetricsNotFoud` was added to highlight that this error coming from Fetch() or GetMetric**s**() and that give a possibility to:
  - add handling an error for ns = "/" - that means that try to get all cataloged metrics returns empty mts, so metric catalog is empty
  - more descriptive error message for requested ns which contains query, e.g. "No metric below the given namespace: /intel/mock/"

- whereas `errorMetricNotFound` is returned by validation that this concrete ns (without query) exists in metric catalog

That is the only difference between them, so**there is a possibility to combined them in one** - I will do that.